homeassistant/components/sma/sensor.py

```diff
@@ -45,6 +813,7 @@ async def async_setup_entry(
             SMAsensor(
                 coordinator,
                 config_entry.unique_id,
+                SENSOR_ENTITIES.get(sensor.name),
```

Comment:
checked lib here why `.key` isn't used as `.key` seems to be `.name` with a number appended if there are multiple